File: LeetCode/Sort the People.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        mapH_N = {}
        
        for i in range(len(names)):
            mapH_N[heights[i]] = names[i]
        
        heights.sort(reverse=True)
        
        sol = []
        
        for i in range(len(heights)):
            sol.append(mapH_N[heights[i]])
            
        return sol
        

File: LeetCode/Sort the People.go
func sortPeople(names []string, heights []int) []string {
    namesToH := make(map[int]string)
    
    for i:=0; i < len(names); i++ {
        namesToH[heights[i]] = names[i]
    }
    
    sort.Sort(sort.Reverse(sort.IntSlice(heights)))
    sol := make([]string, len(names))
    
    for i:=0; i < len(names); i++ {
        sol[i] = namesToH[heights[i]]
    }
     
    return sol
}